src/pdfviewer/PDFDocument.cpp

```diff
@@ -4455,6 +4455,7 @@ void PDFDocument::toggleFullScreen(bool fullscreen)
 	bool presentation = false;
 	if (fullscreen) {
 		// entering full-screen mode
+		wasContinuous = actionContinuous->isChecked() == true;
```

Comment:
it is enough to assign the isChecked state.
If you insist on `== true` , put it into bracket s to improve readability.